src/python/turicreate/_deps/README.md

```diff
@@ -0,0 +1,134 @@
+# Lazy import layout
+
+---
+
+## raitionale
+
+Most times, people tend to perform only one task at one time. There's no need to eagerly import all toolkits when `import turicreate`. When needed, the toolkit will be loaded in runtime
```

Comment:
This kind of omits the main motivation for doing lazy imports -- the speed of `import turicreate`. That is the main reason this work happened, so it should probably be front-and-center in the README.

Reply:
Oh yeah, I forget.